httpgrpc/server/server.go

```diff
@@ -234,3 +239,8 @@ func fromHeader(hs http.Header) []*httpgrpc.Header {
 	}
 	return result
 }
+
+func containsDoNotLogErrorKey(hs http.Header) bool {
```

Comment:
This is too shallow. I understand it's that way for testing, but this functionality can be tested together with the rest of server handler too.

Reply:
I have removed this method, as well as the corresponding test.